stable/metallb/templates/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: metallb-system
```

Comment:
Why do you hard-code the namespace and create it explicitly. Is this a requirement for MetalLB?